### PR TITLE
Test branch megre to main

### PR DESCRIPTION
**Pull request by bhanuPrakashMa**

### CODE REVIEW

File: __init__.py
Comment (Keldami) [inline __init__.py:1]: Please remove this. 

This should not be here. 

The backend project starts at src/backend

Configure your IDE that it starts there.

Comment (Keldami) [inline __init__.py:1]: Please remove this. 

This should not be here. 

The backend project starts at src/backend

Configure your IDE that it starts there.

File: src/backend/requirements.txt
Comment (ingunnaf) [inline src/backend/requirements.txt:1]: Here there's a typo, it should be 
flask_restx==1.2.0
requests==2.31.0

File: __init__.py
Comment (bhanuPrakashMa) [reply __init__.py:1]: please check the changes again

**Review by Keldami — APPROVED** [review]

**Pull request merged**